webui/src/pages/repositories/repository/compare.jsx

```diff
@@ -45,6 +48,12 @@ const CompareList = ({ repo, reference, compareReference, prefix, onSelectRef, o
         await statistics.postStatsEvents(deltaDiffStatEvents);
     }
 
+    const router = useRouter();
+    const handleExchangeRefs = () => {
```

Comment:
While we're changing the terminology...
```suggestion
    const handleSwitchRefs = () => {
```
Not a must. Up to you.